hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/io/erasurecode/TestCodecRawCoderMapping.java

```diff
@@ -150,4 +155,44 @@ public void testIgnoreInvalidCodec() {
             conf, ErasureCodeConstants.XOR_CODEC_NAME, coderOptions);
     Assert.assertTrue(decoder instanceof XORRawDecoder);
   }
+
+  @Test
+  public void testCodecNativeEnabled() {
+    if (!ErasureCodeNative.isNativeCodeLoaded()) {
```

Comment:
This looks more like a use case for JUnit [`Assume#assumeTrue`](https://junit.org/junit4/javadoc/4.13/org/junit/Assume.html#assumeTrue(boolean)).